src/main/java/com/cellar/stickerify/image/ImageHelper.java

```diff
@@ -55,7 +52,7 @@ private static boolean isValidImage(File file) {
 
 		try {
 			String mimeType = new Tika().detect(file);
-			isValid = mimeType.startsWith(MIME_TYPE_IMAGE) && isSupportedImage(mimeType);
+			isValid = isSupportedImage(mimeType);
```

Comment:
This can be moved after the try/catch and you can now return directly `isSupportedImage(mimeType)`